mgr/dashboard_v2: Linting support and tox configuration

rjfd opened the pull request:
This PR adds pylint and pycodestyle support, and also adds a `tox.ini` file to run unit tests and linting which generates coverage information.

Signed-off-by: Ricardo Dias <rdias@suse.com>

rjfd committed:
mgr/openattic: Added pylint support and fixed linting errors

Signed-off-by: Ricardo Dias <rdias@suse.com>

rjfd committed:
mgr/openattic: added tox conf file to run unit tests and linting

Signed-off-by: Ricardo Dias <rdias@suse.com>

votdev commented on src/pybind/mgr/openattic/module.py:
```diff
@@ -9,19 +9,18 @@
+class Module(MgrModule):
     """
-    Hello.
+    openATTIC module entrypoint
```
Should be named 'dashboard' now :-) oA is deprecated :-(

rjfd committed:
msgr/dashboard_v2: Fix python 2/3 compability problems

Signed-off-by: Ricardo Dias <rdias@suse.com>

rjfd commented:
@LenzGr @votdev @sebastian-philipp @ricardoasmarques I've rebased this PR on top of the latest changes and fixed the python 2/3 compatibility problems.
This PR also updates the README with instruction on how to run `tox`.

sebastian-philipp reviewed (approved):
lgtm
